### DISCUSSION TITLE
Any plans to render tiles using the style.json like tileserver-gl?

### DISCUSSION THREAD

**Discussion by donnyv** [maplibre/martin#977] [Q&A] [answered]
I noticed the only project that renders mvt tiles is tileserver-gl. But I'm not really a fan of the performance of nodejs and what it takes to scale it up. I also want the ability to use postgis and dynamically choose tables. Then access with a WMTS layer. I was thinking of building a wrapper around maplibre-native using C#. But its something I'd like to avoid if you guys are planning on adding this feature.

**Comment by nyurik** [comment, marked as answer]
This is definitely a TODO, and I created #978 to track it.  We do need the upstream MapLibre-native project to implement support for this first.

**Reply by donnyv** [reply]
This is great. I will keep an eye on the issue. Thanks!

**Reply by nyurik** [reply]
sure thing, and as always, help is always needed (and welcomed) ;)

**Reply by donnyv** [reply]
I've mainly only done C# on the server. But no reason I can't make this my first Rust project.

**Reply by nyurik** [reply]
be warned, jumping into low level rust-c++ integration is ... tough :)
